comfy_panel/config.lua

```diff
@@ -578,6 +587,30 @@ local build_config_gui = (function(player, frame)
             )
             scroll_pane.add({type = 'line'})
         end
+
+
+        label = scroll_pane.add({type = 'label', caption = 'Map Settings'})
+        label.style.font = 'default-bold'
+        label.style.padding = 0
+        label.style.left_padding = 10
+        label.style.top_padding = 10
+        label.style.horizontal_align = 'left'
+        label.style.vertical_align = 'bottom'
+        label.style.font_color = Color.hot_pink
```

Comment:
```suggestion
        label = scroll_pane.add({type = 'label', caption = 'Map Settings'})
gui_style(label, {
    font = 'default-bold'
    padding = 0
    left_padding = 10
    top_padding = 10
    horizontal_align = 'left'
    vertical_align = 'bottom'
    font_color = Color.hot_pink  
})
```
How about this? ;) With `require` to `utils.utils`

Reply:
I prefer not. I copied this code from another group header. They all should be redone in one function but in a separate PR.